packages/store-api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -0,0 +1,162 @@
+import type { IStoreCart, IStoreOffer } from '../../../__generated__/schema'
+import type {
+  OrderForm,
+  OrderFormItem,
+  OrderFormInputItem,
+} from '../clients/commerce/types/OrderForm'
+import type { Context } from '..'
+
+const getId = (item: IStoreOffer) =>
+  [item.itemOffered.sku, item.seller.identifier, item.price].join('::')
+
+const orderFormItemToOffer = (
+  item: OrderFormItem,
+  index?: number
+): IStoreOffer & { index?: number } => ({
```

Comment:
What do you think about creating an interface (or even type) to reuse this code?
```ts
type IndexedType  = { index?: number }
```
```suggestion
): IStoreOffer & IndexedType => ({
```
Name things isn't one of my best skills, as you can see.